### PR TITLE
Create FunctionScheduler test service

### PR DESCRIPTION
**Pull request by vermakhushboo**
## What does this PR do?

Since scheduled functions take too long, this PR splits functions tests into two services - `Functions` and `FunctionsScheduler` so that they can run parallely.

This reduces total time to run function tests from 21.5m to 11.5m

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work. Screenshots may also be helpful.)

## Related PRs and Issues

- (Related PR or issue)

## Checklist

- [x] Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
- [ ] If the PR includes a change to an API's metadata (desc, label, params, etc.), does it also include updated API specs and example docs?

### CODE REVIEW

File: tests/e2e/Services/FunctionsScheduler/FunctionsCustomClientTest.php [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+use Utopia\Database\Helpers\ID;
+use Utopia\Database\Helpers\Role;
+
+class FunctionsCustomClientTest extends Scope
+{
+    use FunctionsBase;
+    use ProjectCustom;
+    use SideClient;
+
+    #[Retry(count: 2)]
```
Comment (christyjacob4) [inline tests/e2e/Services/FunctionsScheduler/FunctionsCustomClientTest.php:21]: why does this need a retry count? Is this still flaky? If yes we need to understand why

File: tests/e2e/Services/Functions/FunctionsBase.php [hunk 2]
```diff
@@ -18,7 +18,7 @@ protected function packageCode($folder)
     protected function awaitDeploymentIsBuilt($functionId, $deploymentId, $checkForSuccess = true): void
     {
         $startTime = time();
-        $maxWaitTime = 15;
+        $maxWaitTime = 40;
```
Comment (christyjacob4) [inline tests/e2e/Services/Functions/FunctionsBase.php:21]: why did we increase this? Which deployments take longer than 15 seconds?

Comment (vermakhushboo) [reply tests/e2e/Services/Functions/FunctionsBase.php:21]: `Ruby` and `Python` runtime tests for some versions take longer than 15 seconds to build.

File: tests/e2e/Services/FunctionsScheduler/FunctionsCustomClientTest.php (hunk 1 above)
Comment (vermakhushboo) [reply tests/e2e/Services/FunctionsScheduler/FunctionsCustomClientTest.php:21]: Scheduled function sometimes doesn't create execution even after 70s and thus, it is still flaky

**Review by christyjacob4 — APPROVED** [review]

**Pull request merged**